subsystem/operators_test.go

```diff
@@ -245,7 +246,7 @@ var _ = Describe("Operators endpoint tests", func() {
 				swag.String(models.ClusterCPUArchitectureS390x),
 				swag.Bool(false),
 			)
-			Expect(cluster.Payload.CPUArchitecture).To(Equal(models.ClusterCPUArchitectureMulti))
+			Expect(cluster.Payload.CPUArchitecture).To(Equal(models.ClusterCPUArchitectureS390x))
```

Comment:
Why change this?

Reply:
It needs to match the version inside `../docs/hive-integration/crds/clusterImageSet.yaml`, which I upgraded

Reply:
We register a cluster with `s390x` arch, we should get this arch now

Reply:
I'm not sure if this was being done intentionally. If the test wasn't failing before I'm worried it was behavior we were somehow expecting.